tests/test-rest-meta-users-controller.php

```diff
@@ -0,0 +1,1087 @@
+<?php
```

Comment:
I agree and I was thinking about doing that but I wanted to keep things simple for now.